.github/workflows/e2e-test-trusted.yaml

```diff
@@ -1,11 +1,11 @@
 name: "E2E Tests (Trusted)"
 
-on:
-  push:
-    branches: [ "main", "release-**", "workflow/**" ]
-  merge_group:
-    types: [ "checks_requested" ]
-  pull_request:
+# on:
```

Comment:
Are we planing to uncomment this section? Should we put a comment on top of the comment (with a ticket nbr?)

Reply:
No we are planning to remove this eventually.

Reply:
I am trying to maintain the CI for EKS as I need it for reference, but eventually will be removed.

Reply:
FWIW You can disable the workflow at a higher level, allowing you to reactivate it on the go without any code modification.

On the actions page, click on a workflow on the left, then go on the `...` on the right:
<img width="505" alt="Screenshot 2025-04-08 at 11 11 34 AM" src="https://github.com/user-attachments/assets/b304ccdd-43ff-41f0-83db-cc0dace4afd0" />

Reply:
I removed comments and disabled from the workflow page.
I left a comment for the release workflow to avoid rebasing and conflicts as that is managed in the [next PR.](https://github.com/scality/mountpoint-s3-csi-driver/pull/4)
Thanks again for this.